### ISSUE TITLE
stamp_ doesn't produce sidebar as in examples

### ISSUE THREAD

**Issue by mikemahoney218** [wilkelab/cowplot#143]
**Brief description of the problem or desired feature.**
Using `stamp()` and its variants don't produce the characteristic bar on the right hand side of the plot as in the [examples](https://wilkelab.org/cowplot/reference/stamp.html). The examples also seem to reflect old behavior where `theme_cowplot()` would be set as the default theme.

To reproduce, run the code from the online examples:
```{r}
p <- ggplot(iris, aes(x = Sepal.Length, y = Sepal.Width)) +
  geom_point(aes(color = factor(Petal.Width)))

stamp_bad(p + guides(color = "none"))
```
![example](https://user-images.githubusercontent.com/38229299/62801248-50dbb880-bab3-11e9-8edc-30823dec4c35.png)

**Comment by mikemahoney218** [comment]
For comparison, here's the expected behavior pulled from vignettes:

![correct](https://user-images.githubusercontent.com/38229299/62801308-77015880-bab3-11e9-9b8c-9a495a08ac6d.png)

**Comment by clauswilke** [comment]
In some circumstances, the line gets clipped, but it is not clear to me when. I experienced the behavior you describe at some point and then I moved the line a little to the left and that fixed it (see example).

``` r
library(ggplot2)
library(cowplot)
#> 
#> ********************************************************
#> Note: As of version 1.0.0, cowplot does not change the
#>   default ggplot2 theme anymore. To recover the previous
#>   behavior, execute:
#>   theme_set(theme_cowplot())
#> ********************************************************

p <- ggplot(iris, aes(x = Sepal.Length, y = Sepal.Width)) +
  geom_point(aes(color = factor(Petal.Width)))

stamp_bad(p + guides(color = "none"))
```

![](https://i.imgur.com/zspjJsF.png)

<sup>Created on 2019-08-09 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

Could you let me know what graphics device and operating system you're using? Also, as a workaround, you can just copy the `stamp()` code and modify line 35: 
https://github.com/wilkelab/cowplot/blob/597b3ead7d02b9690959902ee645ad970c51741c/R/stamp.R#L35

Instead of `c(.9999999999, .9999999999)`, use `c(0.99, 0.99)` or even `c(0.98, 0.98)` and see if that fixes it.

**Comment by mikemahoney218** [comment]
Windows 10; RStudio tells me it's the null device - how would I go about checking further than that?

**Comment by mikemahoney218** [comment]
`c(0.999, 0.999)` seems to be the sweet spot to get the line on the edge in this example:

![example](https://user-images.githubusercontent.com/38229299/62802463-f3953680-bab5-11e9-8c5c-68365e79e831.png)

**Comment by clauswilke** [comment]
Yeah, so different graphics devices seem to have different sensitivity to where they start clipping. It's frustrating. Maybe I'll make the position configurable.
